packages/alchemy/src/chains.ts

```diff
@@ -11,10 +13,8 @@ import {
   polygon,
   polygonMumbai,
   sepolia,
-  base,
-  baseGoerli,
-  baseSepolia,
-} from "viem/chains";
+} from "@alchemy/aa-core";
+import type { Chain } from "viem";
 
 export const SupportedChains = new Map<number, Chain>([
```

Comment:
do we need this if you have `AlchemyChainMap` in the utils/index.js

Reply:
oh yea good question, lemme export AlchemyChainMap and use that here